pkg/config/config.go

```diff
@@ -31,6 +31,8 @@ import (
 type Config struct {
 	HTTP                    Endpoint          `yaml:"http"`
 	RequestAggregationDelay time.Duration     `yaml:"request_aggregation_delay"`
+	Provider                *string           `yaml:"provider"`
```

Comment:
I would keep it "string", not "*string", as it is expected in most cases.

Reply:
Changed